tests/baselines/reference/dynamicImportWithNestedThis_es2015.js

```diff
@@ -29,8 +29,8 @@ c.dynamic();
             this._path = './other';
         }
         dynamic() {
-            var _a;
-            return _a = this._path, __syncRequire ? Promise.resolve().then(() => require(_a)) : new Promise((resolve_1, reject_1) => { require([_a], resolve_1, reject_1); });
+            var _a, _b;
+            return _a = this._path, __syncRequire ? (_b = _a, Promise.resolve().then(() => require(_b))) : new Promise((resolve_1, reject_1) => { require([_a], resolve_1, reject_1); });

```

Comment:
The additional aliasing here is unnecessary.

Reply:
You're correct. However I'm not sure if it's possible to be fixed on a generic level, since the temp variable created from the UMD transformer may be mutated. I've committed the new baseline, but there are some places where it looks suspicious. I can't really read the output to tell if it's unsafe, so I'll appreciate if you can help to review here.